src/styles/main.scss

```diff
@@ -1,3 +1,70 @@
-body {
+$size: 100px;
+$gap: 1px;
+$day-of-week:
+  mon 0,
+  tue 1,
+  wed 2,
+  thu 3,
+  fri 4,
+  sat 5,
+  sun 6;
+
+* {
   margin: 0;
+  padding: 0;
```

Comment:
Don't reset margins/padding with * selector, reset them only for elements that need it